src/components/toggle/toggle.scss

```diff
@@ -0,0 +1,86 @@
+@import '../../styles/variables';
+
+.kui-theme--light {
+  --color-toggle-on: #{$color-primary-2};
+  --color-toggle-on-bar: #{rgba($color-primary-2, 0.3)};
+  --color-toggle-off: #{$color-bg-dark-9};
+  --color-toggle-off-bar: #{rgba($color-bg-dark-9, 0.3)};
+}
+
+.kui-theme--dark {
+  --color-toggle-on: #{$color-primary};
+  --color-toggle-on-bar: #{rgba($color-primary, 0.3)};
+  --color-toggle-off: #{$color-light};
+  --color-toggle-off-bar: #{rgba($color-light, 0.3)};
+}
+
+.pipeline-toggle {
+  transition: opacity 0.2s ease;
+}
+
+.pipeline-toggle--no-hasCode {
```

Comment:
same as above

Reply:
heyy thanks I will fix this! 

Just a request - if you have already highlighted the problem, then you can avoid 'same as above' comment .. it's just that it creates too much noise when reading the reviews :D

Reply:
totally understand - I added these comments mainly to aid my review as it helps to highlights the areas to address and helps me to track the further updates on github

Reply:
As per @bru5 's comment, I understand about the `pipeline` prefix - this is definitely problematic, but I see that this is out of scope for this work and hence resolving this.